source/features/hide-markdown-diff.tsx

```diff
@@ -1,11 +1,19 @@
 import './hide-markdown-diff.css';
 import React from 'dom-chef';
+import select from 'select-dom';
```

Comment:
(Testing a feature, ignore)
```suggestion
import select  from 'select-dom';
```